bugbug/repository.py

```diff
@@ -51,24 +51,39 @@ def __init__(
         desc,
         date,
         pushdate,
-        bug,
+        bug_id,
         backedoutby,
         author_email,
-        files,
-        file_copies,
         reviewers,
+        files=[],
+        file_copies={},
+        ignored=False,
     ):
         self.node = node
         self.author = author
+        self.bug_id = bug_id
         self.desc = desc
         self.date = date
         self.pushdate = pushdate
-        self.bug = bug
         self.backedoutby = backedoutby
+        self.ever_backedout = backedoutby != ""
```

Comment:
Nit: you could make this a property function

Reply:
I thought about that but since we use this value and need it at the end I thought it was "better" to have it instead of computing it again and again.
But it's as you wish.